From fc760c52ea7877b60ba57c16a3697e9ca524b828 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 24 Oct 2024 20:09:59 +0200
Subject: [PATCH] uboot-mpc85xx: Fix compiling with new u-boot version

Added missing dependency on python-poetry and fix correct name for nor version
of bin file of u-boot.
---
 package/boot/uboot-mpc85xx/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/boot/uboot-mpc85xx/Makefile b/package/boot/uboot-mpc85xx/Makefile
index 916d4ae1cd..d85aafcf75 100644
--- a/package/boot/uboot-mpc85xx/Makefile
+++ b/package/boot/uboot-mpc85xx/Makefile
@@ -8,11 +8,11 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/u-boot.git
-PKG_MIRROR_HASH:=760e620d9d452a46428b539a9a7bc95aa0bae9c42e2c97f1af2f9eae0e668933
+PKG_MIRROR_HASH:=6d3e818b83e932a8e23b8a2fb17482a9c9cdd834350e9688fae5bb29bc356b6f
 PKG_SOURCE_DATE:=2024-05-10
 PKG_SOURCE_VERSION:=cf50dc0956a2b99dff735ca08a6b72446950ea7f
-PKG_BUILD_DEPENDS:=swig/host
-HOST_BUILD_DEPENDS:=swig/host
+PKG_BUILD_DEPENDS:=swig/host python-poetry-core/host
+HOST_BUILD_DEPENDS:=swig/host python-poetry-core/host
 
 include $(INCLUDE_DIR)/u-boot.mk
 include $(INCLUDE_DIR)/package.mk
@@ -36,7 +36,7 @@ define U-Boot/turris1x-nor
   BUILD_DEVICES:=cznic_turris1x
   BUILD_SUBTARGET:=p2020
   UBOOT_CONFIG:=turris_1x_nor
-  UBOOT_IMAGE:=u-boot-with-dtb.bin
+  UBOOT_IMAGE:=u-boot-dtb.bin
 endef
 
 UBOOT_TARGETS:= \
-- 
2.47.0

